src/fluree/db/api/query.cljc

```diff
@@ -326,6 +326,21 @@
                         acc))]
       (vals res))))
 
+#?(:cljs
+   (defn block-Flakes->vector
+     "Convert flakes into vectors.
+     Notes:
+     Cannot use IPrintWithWriter override since calls to storage-handler
+     download blocks using the #Flake format to support internal query
+     handling."
+    [blocks]
+    (loop [[block & r] blocks
+           acc []]
+      (if block
+        (let [flakes (map flake/Flake->parts (:flakes block))]
+          (recur r (into acc [(assoc block :flakes flakes)])))
+        acc))))
+
```

Comment:
This entire function could be accomplished with:
(mapv (fn [block] (assoc block :flakes (mapv vec (:flakes block)))) blocks)

There should be no need to use flake/Flake-parts here - Flake implements iterator/-iseq so you can call `vec` on it.

Reply:
done.